plugin.rb

```diff
@@ -938,10 +939,11 @@ def show
 
     def create
       # guardian.ensure_can_create_explorer_query!
-
+      
       query = DataExplorer::Query.from_hash params.require(:query)
-      query.created_at = Time.now.strftime("%b %e, %Y")
+      query.created_at = Time.now
       query.created_by = current_user.id.to_s
+      query.last_run_at = '-'
```

Comment:
Feels weird. Why isn't this done in the client-side? This looks like a presentation-only edge case.

Reply:
Yeah my bad, defaulting this to `Time.now` makes more sense than this. Thanks!!